app/utils.py

```diff
@@ -133,3 +133,14 @@ def update_dct_to_str(update_dct):
         str += "- {}".format(key.replace("_", " "))
         str += "\n"
     return str
+
+
+def get_csv_max_rows(service_id):
+    bulk_sending_services = [
+        current_app.config['HC_EN_SERVICE_ID'],
```

Comment:
Does this fail gracefully or throw an error if these vars don't exist?

Reply:
Good point - I will add default values for these.